File: rust/project_084/main.rs
use csv::Writer;
use serde::Serialize;

#[derive(Serialize)]
struct Record {
    name: String,
    age: u8,
}

fn main() {
    let mut wtr = Writer::from_path("output.csv").unwrap();
    let records = vec![
        Record { name: "Alice".to_string(), age: 30 },
        Record { name: "Bob".to_string(), age: 25 },
    ];
    for rec in records {
        wtr.serialize(rec).unwrap();
    }
    wtr.flush().unwrap();
    println!("Wrote output.csv");
}

File: python/project_084/main.py
import csv

filename = input("Enter CSV filename: ")
fields = ['name', 'age', 'email']
data = [
    {'name': 'Alice', 'age': 30, 'email': 'alice@example.com'},
    {'name': 'Bob', 'age': 25, 'email': 'bob@example.com'},
    {'name': 'Charlie', 'age': 35, 'email': 'charlie@example.com'}
]
with open(filename, 'w', newline='') as csvfile:
    writer = csv.DictWriter(csvfile, fieldnames=fields)
    writer.writeheader()
    writer.writerows(data)
print(f"CSV file '{filename}' written with sample data.")
